test/support/resources.ex

```diff
@@ -20,6 +20,23 @@ defmodule LiveAdminTest.User do
   def failing_action(_, _), do: {:error, "failed"}
 end
 
+defmodule LiveAdminTest.CustomStringType do
+  use Ecto.Type
+  @behaviour LiveAdmin.Type
```

Comment:
this could also be done as a protocol instead of a behavior
```
defprotocol LiveAdmin.TypeHandler do
  @doc """
  Returns how a custom type should be rendered in a form field.
  """
  def render_as(type)
end

defimpl LiveAdmin.TypeHandler, for: MyApp.CustomDateType do
  def render_as(_type), do: :string
end
```

But it looked nicer to me having the method on the ecto type instead of separate